src/pydata_sphinx_theme/toctree.py

```diff
@@ -149,7 +152,7 @@ def generate_header_nav_html(n_links_before_dropdown: int = 5) -> str:
             out += f"""
             <li class="nav-item dropdown">
                 <button class="btn dropdown-toggle nav-item" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-controls="pst-header-nav-more-links">
-                    More
+                    {dropdown_text}
```

Comment:
Can we include this in a translation wrapper ? So that translated documentation could override "More" or any other word selected

Reply:
Tried to add `"{{ _(\"Text\") }}"` syntax in here, but it doesn't work and I can't find good examples of translation wrappers in `.py` files. Can you help me a bit here please?

Reply:
isn't `{{ _(dropdown_text) }}` sufficient ?

Reply:
If I use `{{ _({dropdown_text}) }}` I get:

![Screenshot 2023-08-29 at 12 26 24](https://github.com/pydata/pydata-sphinx-theme/assets/16418547/eea1529a-c133-4fc2-aa71-c17e8513671e)

and can't seem to find a way to get this read correctly.

Reply:
I think I did, but will try again 👍

Reply:
No luck:


<img width="578" alt="Screenshot 2023-09-01 at 14 29 19" src="https://github.com/pydata/pydata-sphinx-theme/assets/16418547/abc9d66f-50a5-4708-9d25-1d460547aa17">

Reply:
Gentle ping =)

Reply:
I'm completely  swamped with the refactoring of another project for work. Sorry for the delay

Reply:
I think the problem here is that the *entire* nav bar HTML is being dynamically generated within a Jinja `{{ ... }}` template block, and then we're trying to add another nested `{{ ... }}` template inside it as part of the inserted Python string (which won't work unless Jinja did 2 passes over the document).  One possible solution is to have Jinja if-else blocks in `navbar-nav.html` and do different things depending on whether we need a dropdown or not, and use Jinja `include` to insert a new Jinja template that had the dropdown's HTML (including the translation string) *in HTML* (instead of in a Python string).

Reply:
Somewhere in the back of my mermory I remember ancient runes talking about Sphinx. It was mentioning that you can call the translation dictionary directly from python as the lang is contained in the build context. But the page of this ancient manuscript is now tarnished by the centuries and I need to go down to Sphinx code itself to get this information.

Reply:
I think I fixed it, see [168b212](https://github.com/pydata/pydata-sphinx-theme/pull/1423/commits/168b212e89e9454a730e29a2d7b95c6ed56f1694)

Reply:
yep good to go for me !